### DISCUSSION TITLE
minor issue

### DISCUSSION THREAD

**Discussion by VikingTechLLC** [Comfy-Org/ComfyUI-Manager#54] [General]
Just noticed in the terminal a minor spelling error with the word "fetch".

That's all.  Thanks
![image](https://github.com/ltdrdata/ComfyUI-Manager/assets/84103587/cce1c825-fb06-4305-a94b-4f8f27dc3cb1)

**Comment by ltdrdata** [comment]
Fixed. Thank you.
